demos/next/src/pages/_app.jsx

```diff
@@ -0,0 +1,28 @@
+import Head from 'next/head'
+import { useRouter } from 'next/router'
+import useAckee from 'use-ackee'
+
+import Nav from '../components/nav'
+
+const title = 'useAckee with Next.js'
+
+const UseAckeeApp = ({ Component, pageProps }) => {
+  const { pathname } = useRouter()
+
+  useAckee(pathname, {
+    domainId: 'hd11f820-68a1-11e6-8047-79c0c2d9bce0',
+    server: 'https://example.com',
+  })
```

Comment:
The original demo had `ignoreLocalhost: false` which resulted in CORS errors (and would otherwise fail even without them). Without that setting (which you probably don't recommend for real apps anyway), you get the nice console.info which shows that Ackee is successfully running. Happy to revert if you prefer, though.